src/program/lwaftr/run_nohw/README

```diff
@@ -0,0 +1,16 @@
+Usage: run-nohw --help
+       run-nohw --conf PATH --b4-if NAME --inet-if NAME [--verbose]
+
+       --conf, -c PATH      Path to the lwAFTR configuration file.
+       --b4-if, -B NAME     Name of the B4-side network interface.
+       --inet-if, -I NAME   Name of the Internet-side network interface.
+       --verbose, -v        Be verbose. Can be used multiple times.
+       --csv-file FILENAME
```

Comment:
It should not be called 'csv-file'; the name should describe the contents, not the filetype.